README.md

```diff
@@ -1,5 +1,5 @@
-About python-blosc
```

Comment:
I guess that is is because of the rerendering of the feedstock? Maybe you need to rerender again.